newsfragments/1495.misc.rst

```diff
@@ -0,0 +1 @@
+Add type hints to ENS module.
```

Comment:
I don't think we need newsfragments for these.  Probably only need a fragment at the point where we enable type hints for 3rd party installers of the web3 library.